.github/workflows/nx-cloud-main.yml

```diff
@@ -139,7 +139,7 @@ jobs:
           if [[ $pnpm_ver != '' ]]; then echo "PNPM: $pnpm_ver"; fi
 
           echo "node_version=${node_ver:1}" >> $GITHUB_OUTPUT
-          echo "yarn_version=${yarn_ver:1}" >> $GITHUB_OUTPUT
+          echo "yarn_version=${yarn_ver}" >> $GITHUB_OUTPUT
```

Comment:
Obviously we can't move forward with this because it's causing the integration tests to consistently fail.

I am not sure why, but more broadly than that this has highlighted that I don't what this is actually doing...

I had assumed this was setting a default value of `1` in the case that `yarn_ver` was not set, but it seems that the syntax for that would be `:=` or `:-`, not just `:`

So @meeroslav @rrooding what is this code doing before and after please?

Reply:
It's removing the first character using Bash substring expansion:
https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html
```
${parameter:offset}
${parameter:offset:length}
  This is referred to as Substring Expansion. It expands to up to length characters of the value of parameter starting at the character specified by offset.
```

If I have time I will look into the test this week but I might need some help there. I have the suspicion that the test was already incorrect anyway because it didn't really test the yarn / yarn berry stuff anyway.

Reply:
Thanks for the clarification @rrooding.

For the integration tests, I've figured out it's down to github blocking the sharing of the `secrets` namespace with forks. I've pushed a fallback value for this and synced the branch so fingers crossed it will be green now and we can merge it